modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PageMemoryImpl.java

```diff
@@ -1212,7 +1219,11 @@ private boolean copyPageForCheckpoint(
         try {
             long tmpRelPtr = PageHeader.tempBufferPointer(absPtr);
 
-            boolean success = clearCheckpoint(fullId);
+            boolean pageCopyed = false;
+
+            byte checkpointIdx = (byte)((tmpRelPtr & CHECKPOINT_INDEX_MASK) >> 56);
```

Comment:
I would prefer to have an appropriate method for that operation. I mean `getCheckointId(long tmpRelPtr)`